honeybee_radiance_command/rpict.py

```diff
@@ -0,0 +1,146 @@
+"""rpict command."""
+
+from honeybee_radiance.view import View
+from .options.rpict import RpictOptions
+from ._command import Command
+import honeybee_radiance_command._exception as exceptions
+import honeybee_radiance_command._typing as typing
+
+
+class Rpict(Command):
+    """rpict command."""
+
+    __slots__ = ('output_name', 'octree', 'view', 'sky_file', 'simulation_type')
```

Comment:
This is becoming similar to what we did in honeybee[+] which I think was wrong. We should keep the inputs structure clean. See how it is done in rtrace: https://github.com/ladybug-tools/honeybee-radiance-command/blob/master/honeybee_radiance_command/rtrace.py